front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1257,6 +1257,8 @@ export type RollingStockBase = {
     [key: string]: string;
   };
   energy_sources: EnergySource[];
+  electrical_power_startup_time?: number;
```

Comment:
If you add the fields to the required list, then they won't be potentially undefined in the frontend ;) 
```suggestion
  electrical_power_startup_time: number;
```